docs/Deployment.md

```diff
@@ -112,38 +112,16 @@ Log transports are where the logs are written to. If this isn't provided in the
 ## AWS Lambda
 
 1. Run `npm run zip-aws-lambda` from the [repository root directory](../extension), to zip the contents in preparation for uploading to AWS
-2. An AWS lambda function should be created ([Guide to creating lambda functions](https://docs.aws.amazon.com/lambda/latest/dg/getting-started-create-function.html)). The runtime should be Node.js 14.x.
+2. An AWS lambda function should be created ([Guide to creating lambda functions](https://docs.aws.amazon.com/lambda/latest/dg/getting-started-create-function.html)). The runtime should be Node.js, recommended version is 18.x.
```

Comment:
[aws supports woohooo](https://aws.amazon.com/blogs/compute/node-js-18-x-runtime-now-available-in-aws-lambda/) 🥳